aspnetcore/fundamentals/w3c-logger/samples/6.x/Startup.cs

```diff
@@ -0,0 +1,50 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.HttpLogging;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+
+namespace W3CLoggerSample
+{
+    public class Startup
+    {
+        #region configureservices
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.AddW3CLogging(logging =>
+            {
+                // Log all W3C fields
+                logging.LoggingFields = W3CLoggingFields.All;
+                logging.FileSizeLimit = 5 * 1024 * 1024;
```

Comment:
Minor nit: consider adding a newline after

```C#
// Log all W3C fields
logging.LoggingFields = W3CLoggingFields.All;
```

That makes it clearer that the comment applies to just that line.